tests/acceptance/test_emails.py

```diff
@@ -37,17 +37,9 @@
 
 
 def read_txt_email_fixture(name):
-    # XXX(python3): We have different fixtures for python2 vs python3 tests
-    # because of the differences in how the random module works betwen 2 and 3.
-    # NOTE that we _cannot_ just set the version of the seed, as there are more
-    # differences. See [0].
-    #
-    # [0]: https://stackoverflow.com/questions/55647936/random-randint-shows-different-output-in-python-2-x-and-python-3-x-with-same-see/55648073
-    version_suffix = "_py2" if six.PY2 else ""
-
     # "sso unlinked without password"
     # => "sso_unlinked_without_password.txt"
-    filename = name.replace(" ", "_") + version_suffix + ".txt"
+    filename = name.replace(" ", "_") + ".txt"
```

Comment:
Gotta delete all these test fixtures now too

Reply:
Ah that's a good catch.